### ISSUE TITLE
Allow user to control frequency of reporting training loss

### ISSUE THREAD

**Issue by wsnoble** [Noble-Lab/casanovo#585]
We should add to the config file an option called `train_check_interval` that is analogous to the existing `val_check_interval`. Then do the loss reporting by step instead of by epoch in model.py and give this config option to the Trainer in model_runner.py.

**Comment by mhmdkardosha** [comment]
Hi @wsnoble, I would love to work on this issue, and I will open a PR for it now.
